schedule/ha/bv/migration/migration_offline_sles_ha.yaml

```diff
@@ -67,6 +68,18 @@ conditional_schedule:
     ARCH:
       s390x:
         - installation/bootloader_zkvm
+  grub_test:
+    ARCH:
+      aarch64:
+        - installation/grub_test
+      ppc64le:
+        - installation/grub_test
+      x86_64:
+        - installation/grub_test
+  mgmt_console:
+    ARCH:
+      s390x:
+        - boot/reconnect_mgmt_console
```

Comment:
I think you can merge `grub_test` and `mgmt_console` into a single conditional update as both are deciding base on the **ARCH** setting and one comes right after the other.

Reply:
Yes, I thought to do that at first and I said to myself that is better to have two different names.
But yeah, I can change it.